forum/qa-include/qa-index.php

```diff
@@ -165,11 +165,13 @@ function qa_index_set_request()
 
 		$requestlower = strtolower(qa_request());
 
-		if ($requestlower == 'install')
-			require QA_INCLUDE_DIR.'qa-install.php';
-		elseif ($requestlower == 'url/test/'.QA_URL_TEST_STRING)
-			require QA_INCLUDE_DIR.'qa-url-test.php';
-		else {
+		if ($requestlower == 'install') {
+			require QA_INCLUDE_DIR . 'qa-install.php';
+		} elseif ($requestlower == 'url/test/'.QA_URL_TEST_STRING) {
+			require QA_INCLUDE_DIR . 'qa-url-test.php';
+		} elseif ($requestlower == 'blocked-users-list') {

```

Comment:
poszukaj sposobu na nadpisanie tego bez nadpisywania core q2a. Wolałbym to zmienić w pluginie.

Reply:
To akurat da się zrobić w plugnie (w sensie custom page) ale tak jak wspominałem wyżej nie ma to tutaj sensu.

Reply:
jeżeli da się to zrobić w pluginie, to bezapelacyjnie musi być w pluginie